packages/date-picker/src/vaadin-date-picker-light.js

```diff
@@ -161,6 +132,17 @@ class DatePickerLight extends ThemableMixin(DatePickerMixin(ValidateMixin(Polyme
     this.$.overlay.positionTarget = this.inputElement;
     this.$.overlay.noVerticalOverlap = true;
   }
+
+  /** @private */
+  __updateOverlayTheme(overlayContent, theme) {
```

Comment:
Does it make sense to move this to `vaadin-date-picker-mixin.js` as well, as it's duplicated in `vaadin-date-picker`?

Reply:
Done.